pkg/sql/jobs/jobs.proto

```diff
@@ -29,8 +30,19 @@ message RestoreDetails {
   bytes low_water_mark = 1;
 }
 
+message ResumeSpanList {
+  repeated roachpb.Span resume_spans = 1 [(gogoproto.nullable) = false];
+}
+
 message SchemaChangeDetails {
- util.hlc.Timestamp read_as_of = 1 [(gogoproto.nullable) = false];
+  util.hlc.Timestamp read_as_of = 1 [(gogoproto.nullable) = false];
+  // A schema change can involve running multiple processors backfilling
+  // or deleting data. They occasionally checkpoint Spans so that the
+  // processing can resume in the event of a node failure. The spans are
+  // non-overlapping contiguous areas of the KV space that still need to
+  // be processed.
+  repeated ResumeSpanList resume_span_list = 2 [(gogoproto.nullable) = false];
```

Comment:
I believe the index here represents the index of a mutation with the same mutation id?

So a job represents many backfill operations with the same mutation id.